server/db/database.js

```diff
@@ -35,11 +35,13 @@ const versions = [
  */
 export const currentAPIVersion = () => versions[versions.length - 1];
 
+const { NODE_ENV, DB_DATABASE, DB_DATABASE_TEST, DB_HOST, DB_PASSWORD, DB_USER } = process.env;
```

Comment:
1/5 As we don't reuse them later, I don't think it's useful to create new `const` with theses values.

Reply:
As you want, it was juste to avoid writing 5 times `process.env` to be a bit clearer about the database connection.

Reply:
What does @valentinperignon think about that ?

Reply:
I agree with @FrancoisPog (please don't leave the project)